### DISCUSSION TITLE
Max. File Size for Compression

### DISCUSSION THREAD

**Discussion by Nisargparekh7** [Dr-Emann/applesauce#38] [General]
What is the maximum file size does applesauce can compress?

**Comment by Dr-Emann** [comment]
Applesauce can compress files up to 4 GiB (`4294967296 bytes`). This is a macos limitation: larger files when compressed cause [kernel panics](https://github.com/RJVB/afsctool/issues/17#issuecomment-1296522656) when read.

**Reply by Nisargparekh7** [reply]
And any future plans to increase this max. size

**Reply by Dr-Emann** [reply]
This is a limitation in macos, unfortunately there's nothing I can do.

**Reply by Nisargparekh7** [reply]
Okay, Thank you
